src/problem-analysis/target-group-analysis.tex

```diff
@@ -21,21 +24,25 @@ \section{Target group analysis}\label{sec:target-group-analysis}
 The Figure~\ref{fig:graph} shows a graph made to illustrate different skill levels amongst the user base
 of \url{Chess.com}.
 The blue wave represents the amount of users with a certain rating.
-We don't have direct numbers as the graph is very limited in terms of data.
+The team doesn't have direct numbers as the graph is very limited in terms of data.
 The color bars on the bottom represent different skill groups~\cite{chess-ratings}.
 
 The first group, which is marked in green and is the biggest one, is novices.
-They are not expected to know much about chess or its rules, why they are not fit to be our target group, as our
+They are not expected to know much about chess or its rules, why they are not fit to be the target group, as the
 goal is not to teach the rules of chess, but rather to help people improve their skills.
 The second group, which is marked in yellow, is beginners.
-This is our main target group, as they are the ones that are expected to benefit the most from our product.
-The graph peaks at the end of the novices group, which means that transitioning from novice to beginner seems to be the
-hardest challenge for new players.
+This is the main target group, as they are the ones that are expected to benefit the most from the product.
+That's because the graph peaks at the end of the novices group, which means that transitioning from novice to beginner
+seems to be the hardest challenge for new players.
+Which is why the team has decided to focus on this group; to help them overcome this obstacle.
+By focusing on this group, the team hopes to flatten the curve and make the transition from beginner to intermediate
+easier.
 The third and fourth groups, which are marked in light and dark orange, are intermediate 1 and 2 players.
-While they are not our main target group, they may still have a subset of the problems that the beginners have.
+While they are not the main target group, they may still have a subset of the problems that the beginners have.
+Therefore, they can still benefit from the product.
```

Comment:
As Andres said, we should not speak of the "product" in the problem analysis section. Consider rewriting this sentence